tests/console/yast2_dnsserver_service_active_disabled.pm

```diff
@@ -0,0 +1,35 @@
+# SUSE's openQA tests
```

Comment:
the name of the test should represent the action not the validation... looks like we just stop the service, we don't touch the 'after_reboot' in the original case.
shouldn't be this then `yast2_dns_server_stop` (we can omit service).

Reply:
For this test it set 'start' & 'do not start', if you'd like the name represent the action, then the name can change to yast2_dns_server_start_do_not_start ?